src/EditorFeatures/VisualBasicTest/CodeActions/MoveType/BasicMoveTypeTestsBase.vb

```diff
@@ -0,0 +1,90 @@
+' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Xml.Linq
+Imports Microsoft.CodeAnalysis.CodeRefactorings
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.MoveType
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces
+Imports Microsoft.CodeAnalysis.VisualBasic.CodeRefactorings.MoveType
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.CodeRefactorings.MoveType
+    Public Class BasicMoveTypeTestsBase
+        Inherits AbstractMoveTypeTestsBase
+
+        Protected Overrides Function CreateCodeRefactoringProvider(workspace As Workspace) As CodeRefactoringProvider
+            Return New MoveTypeCodeRefactoringProvider()
+        End Function
+
+        Protected Overrides Function CreateWorkspaceFromFileAsync(
+            definition As String,
+            ParseOptions As ParseOptions,
+            CompilationOptions As CompilationOptions
+        ) As Task(Of TestWorkspace)
+
+            Return TestWorkspace.CreateVisualBasicAsync(
+                definition,
+                ParseOptions,
+                If(CompilationOptions, New VisualBasicCompilationOptions(OutputKind.DynamicallyLinkedLibrary)))
```

Comment:
Why do you need this?  Won't CreateVisualBasicAsnyc handle the case where it gets null options?